frontend/src/components/form/base/EDatePicker.vue

```diff
@@ -51,38 +52,89 @@ export default {
   props: {
     value: { type: [String, Number], required: true },
     icon: { type: String, required: false, default: 'mdi-calendar' },
-    valueFormat: { type: [String, Array], default: 'YYYY-MM-DD' }
+
+    // format in which the `value` property is being provided & input events are triggered
+    valueFormat: { type: [String, Array], default: 'YYYY-MM-DD' },
+
+    // v-date-picker allowedDates
+    allowedDates: { type: Function, default: null }
+  },
+  data () {
+    return {
+      // Day.js object of parsed value
+      dateTime: null
```

Comment:
Hier wird der state von this.value des BasePickers dupliziert, aber soweit ich sehe nicht verwendet.
Was war der grund dafür?

Reply:
Ich hab das mehr oder weniger 1:1 vom ETimePicker übernommen.

Dort wird das dateTime object verwendet um via `setTime` jeweils die Zeit zu überschreiben, nicht aber das Datum.

Hier habe ich analog die Funktionalität benötigt das Datum zu überschreiben, nicht aber die Zeit (wenn denn gesetzt).  

Kann ich aber beide refactorn, so dass dateTime nicht als Zustand gespeichert werden muss.

Reply:
Here we go:
https://github.com/ecamp/ecamp3/pull/2366/commits/a168d8be67dfcad963e63aa1f6bfa753b94f14a9

Reply:
danke vielmals